_posts/2016-01-02-git-and-sensitive-data.markdown

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "Protect your sensitive data"
```

Comment:
@schmierkov do you think we could maybe change the title to something more specific, like what you have in the file name? Something like:
- `Git and sensitive data`
- `Protect your sensitive data in Git`
- `How to protect your sensitive data in Git`